mssql-plugin/src/test/java/io/cdap/plugin/mssql/SqlServerConnectorTest.java

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright © 2021 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package io.cdap.plugin.mssql;
+
+import io.cdap.plugin.db.connector.DBSpecificConnectorBaseTest;
+import org.junit.Test;
+
+import java.io.IOException;
+
+
```

Comment:
nit - extra line

Reply:
You also need to have some javadoc on these public classes, I think checkstyle file is missing in this repo

Reply:
checkstyle is not missing in this repo. Test class doesn't require javadoc. Removed the extra line.